pages.pt_BR/linux/abbr.md

```diff
@@ -0,0 +1,21 @@
+# abbr
+
+> Gerencie abreviaçoes para fish-shell.
+> Palavras definidas pelo usuário são substituídas por frases longas assim que são digitadas.
+> Mais informações: <https://fishshell.com/docs/current/cmds/abbr.html>.
+
+- Adicione uma nova abreviação:
+
+`abbr --add {{abbreviation_name}} {{command}} {{command_arguments}}`
```

Comment:
Please translate the placeholders for get this easier for non-English speakers, e.g. `command` to `comando`.